packages/svelte2tsx/test/helpers.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs");
+const assert = require("assert");
+function benchmark(fn) {
+	return -Date.now() + (fn(), Date.now());
+}
+function readFileSync(path) {
+	return fs.existsSync(path) ? fs.readFileSync(path, "utf-8").replace(/\r\n/g, "\n").replace(/\s+$/, "") : null;
+}
+function check_dir(path, { allow, require = allow }) {
+	if (allow !== require) allow = [...require, ...allow];
```

Comment:
I prefer these to have a separate line.
```suggestion
	if (allow !== require) {
		allow = [...require, ...allow];
	}
```